src/Data/Vec/Base.agda

```diff
@@ -263,16 +263,13 @@ splitAt : ∀ m {n} (xs : Vec A (m + n)) →
           ∃₂ λ (ys : Vec A m) (zs : Vec A n) → xs ≡ ys ++ zs
 splitAt zero    xs                = ([] , xs , refl)
 splitAt (suc m) (x ∷ xs)          with splitAt m xs
-splitAt (suc m) (x ∷ .(ys ++ zs)) | (ys , zs , refl) =
-  ((x ∷ ys) , zs , refl)
+splitAt (suc m) (x ∷ xs) | (ys , zs , p) = ((x ∷ ys) , zs , cong (x ∷_) p)
```

Comment:
Suggest `eq` for the proof of the equality, rather than `p`?

Reply:
Suggest also using an irrefutable `with` on LHS, or even a pattern-matching `let` on RHS...

Reply:
both done.